<HTML><HEAD><TITLE> BayesPI2</TITLE> 

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-55609707-3', 'auto');
  ga('send', 'pageview');

</script>
</HEAD>
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-QT6XVRRRL6"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-QT6XVRRRL6');
</script>
<BODY BGCOLOR="FFFFF0"><b></b> 
<h2>
<a href="">
Quality verse accuracy: Result of a reanalysis of protein-binding microarray from the DREAM5 challenge by using BayesPI2 including dinucleotide interdependence
<p></A></b> </h2> 
<br> 

<b><A HREF="mailto:junbai.wang@rr-research.no">Junbai Wang</a>* </b>
<br><br> 
<b>
*Department of Pathology<br>
<br>
Oslo University Hospital - The Norwegian Radium Hospital
<br>
Montebello 0310 Oslo
<br>
Norway

</b> <p> <br>
Abstract
<br>
<br>
Computational modeling transcription factor (TF) sequence specificity is an important research topic of regulatory genomics. Though a systematic comparison of 26 algorithms to learn TF-DNA binding specificity in in vitro protein-binding microarray (PBM) data was published, the quality of those examined PBMs is not evaluated completely. This may have a significant impact on algorithm performance evaluation. Here, new quality-control parameters such as principal component analysis (PCA) ellipse is proposed to assess the data quality of both single PBM and paired PBMs. Additionally, a biophysical model of TF-DNA interactions including adjacent dinucleotide interdependence was implemented in a C program â€“ BayesPI2, where sparse Bayesian learning and the relevance vector machine are used to predict unknown model parameters. In this study, 66 mouse TFs from the DREAM5 challenge were classified into two groups (i.e. good data quality versus bad data quality) based on paired PBM quality-control parameters. Then, algorithm performance of computational methods (i.e. BayesPI2) to model TF sequence specificity was evaluated between the two groups. Results show that both the algorithm performance and the predicted TF-binding energy-level of a motif are significantly influenced by PBM data quality, and the poor PBM data quality is linked to specific protein domain such as C2H2 DNA-binding domain. In particular, the new BayesPI2 dinucleotide energy-dependent model offers great improvement in testing prediction accuracy over the simple energy-independent model, for at least 21% of analyzed TFs, which is almost two times more than that of the original study. 
<br>
</br>
</body>
</html>

